resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
+  },
+
+  "event": {
+	"page_id": "", // ID of the page on which this event happened.
+	"parent_id": "", // Parent event ID if this is a child event.
```

Comment:
As we discussed earlier today, Origami has a big issue with parent events.  We don't like it conceptually: the 'parent' is actually describing an event that happened at a different point in time, and no developer will understand that intuitively.  Origami wants to create technology that people understand how to use intuitively so you don't have to 'wrap your head around it' as a prerequisite to using it correctly.

Separately, implementation is complex, because it requires disparate parts of the page code to maintain shared state, which is an inherently bad pattern, and one that will also lead to memory leaks.

We proposed a system of 'collation ids' for an event, so that instead of passing a parent, we can associate multiple events together in a group, eg:

```
{
  "collation_ids": [
    "131bec90-055e-11e5-b939-0800200c9a66",
    "131c13a1-055e-11e5-b939-0800200c9a66"
  ]
}
```